index.html

```diff
@@ -1,59 +1,425 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Harshika's Portfolio</title>
-    <link rel="stylesheet" href="style.css" class="css">
-    <link href="https://fonts.googleapis.com/css2?family=Inter:wght@800&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
-    <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Harshika's Portfolio</title>
+  <link rel="stylesheet" href="style.css" class="css">
+  <link rel="stylesheet" href="https://kit.fontawesome.com/3ab8daaa6e.css" crossorigin="anonymous">
+  <script src="https://kit.fontawesome.com/3ab8daaa6e.js" crossorigin="anonymous"defer></script>
 </head>
 <body>
-    <header>
+  <!---Navigation-->
+  <section class="imagecontainer">
+ <header>
```

Comment:
- [ ] Please make sure the hamburger remains in the same place as I resize the page.

## FIgma Design
![image](https://user-images.githubusercontent.com/92215878/220710658-a18e3bb4-4189-4a2b-9a93-5510902a1313.png)

## Your Design
![image](https://user-images.githubusercontent.com/92215878/220710767-b9f78917-637e-4edd-8e19-bfea5fbec5ba.png)